### PR TITLE
Change to cope with changes on server.

### PR DESCRIPTION
**Pull request by pedrovelho**
Stop working because now the REST call returns success from the server side triggering the right callback on the javascript.

### CODE REVIEW

File: app/scripts/proactive/rest/studio-client.js [hunk 1]
```diff
@@ -80,7 +80,10 @@ define(
                     type: 'POST',
                     success: function(data) {
                         // ProActive Studio login request return invalid json with status code 200
-                        console.log("Should not be there", data)
+                        that.alert("Connected", "Successfully connected user", 'success');
```
Comment (chloegugli) [inline app/scripts/proactive/rest/studio-client.js]: This alert might be annoying

File: app/scripts/proactive/rest/studio-client.js [hunk 2]
```diff
@@ -80,7 +80,10 @@ define(
...
+                        console.log("Session ID is " + data.responseText)
+                        localStorage['pa.session'] = data.responseText;
```
Comment (chloegugli) [inline app/scripts/proactive/rest/studio-client.js]: This function says (in the comment that it sets localStorage['pa.login']. Why setting localStorage['pa.session'] here? Moreover, you use it in the request header.

**Comment by amouhoub** [comment]
@pedrovelho what about this PR ?

**Comment by pedrovelho** [comment]
I belive this was not to blame at the end for the problem! The problem was some guys that had a plugin installed in their browsers: does it ring a bell?

**Pull request closed without merging**